ykman-gui/qml/LegacyInterfaceErrorPopup.qml

```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+
+InlinePopup {
+
+    Heading2 {
+        width: parent.width
+        text: qsTr("Error!
+
+Failed to configure interfaces.
+
+Make sure the YubiKey does not have restricted access.")
```

Comment:
I've had better results with multi-line messages like this when placing them in separate `Heading2` containers - QML doesn't seem to respect these line breaks.

Reply:
Good point, did a fix for this. We should probably go through all popups later to make sure they look and feel the same.

Reply:
Sounds good!